frontend/packages/ux-editor/src/components/config/editModal/EditImage/ExternalImage/ExternalImage.tsx

```
@@ -88,5 +65,47 @@ export const ExternalImage = ({
+  existingImageUrl: string;
+  onBlur: (event: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+const EditUrl = ({ url, existingImageUrl, onBlur }: EditUrlProps): React.ReactElement => {
+  const { t } = useTranslation();
+  const [isViewMode, setIsViewMode] = useState<boolean>(false);
+  const label = t('ux_editor.properties_panel.images.enter_external_url');
+  const noUrlText = t('ux_editor.properties_panel.images.external_url_not_added');
+  const value = calculateViewValue(url, noUrlText, isViewMode);
+
+  debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)